plugin.yml

```diff
@@ -57,5 +57,7 @@ configuration:
       properties:
         command:
           type: string
+    download:
```

Comment:
I am thinking maybe we should call this option `downloadBinary` to be more descriptive.